### PR TITLE
[SecurionPay] Implemented `void/2`

### PR DESCRIPTION
**Pull request by xC-Stony**
`void/2`
---------
* Cancels a pre-authorized charge(transaction).
* SecurionPay does not support partial cancellation.

Tests
------
* Added integration tests for `void/2` using Exvcr.

Docs
-----
* Mentioned the expiration of stand-alone authorized transaction
* Added docs for `void/2`
* Marked `capture/2` and `void/2` as implemented in module docs

### CODE REVIEW

File: lib/gringotts/gateways/securion_pay.ex [hunk 1]
```diff
@@ -94,6 +94,25 @@ defmodule Gringotts.Gateways.SecurionPay do
     commit(params, "charges", opts)
   end
 
+  @doc """
```
Comment (oyeb) [inline lib/gringotts/gateways/securion_pay.ex]: What happens to authorizations that are not captured (for a long time), do they expire?

Comment (RajeetKaushal) [reply lib/gringotts/gateways/securion_pay.ex]: Sir, they are removed after the authorization expires ( 5 days).

Comment (oyeb) [reply lib/gringotts/gateways/securion_pay.ex]: Please document this in `authorize/3` docs.

Comment (RajeetKaushal) [reply lib/gringotts/gateways/securion_pay.ex]: Okay Sir

**Pull request merged**